src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/register_app_interface_request_test.cc

```diff
@@ -462,6 +464,8 @@ TEST_F(RegisterAppInterfaceRequestTest,
   EXPECT_CALL(app_mngr_, application_by_policy_id(kAppId))
       .WillRepeatedly(Return(mock_app));
 
+  EXPECT_CALL(*mock_app, app_id()).WillOnce(Return(kConnectionKey));
```

Comment:
@ValeriiMalkov I would say that it is the ON_CALL for existing test cases, and you bug fix required ti create 2 additional unit tests